Source/Core/VideoCommon/FramebufferManager.cpp

```diff
@@ -258,8 +258,7 @@ AbstractTexture* FramebufferManager::ResolveEFBColorTexture(const MathUtil::Rect
 AbstractTexture* FramebufferManager::ResolveEFBDepthTexture(const MathUtil::Rectangle<int>& region,
                                                             bool force_r32f)
```

Comment:
The `force_r32f` parameter is now unused; either it can go away as well or we're still missing something